src/pytest_benchmark/stats.py

```diff
@@ -206,6 +207,7 @@ def as_dict(self, include_data=True, flat=False, stats=True, cprofile=None):
             "fullname": self.fullname,
             "params": self.params,
             "param": self.param,
+            "extra_info": self.extra_info,
```

Comment:
I used `extra_info` for consistency with `machine_info`, `commit_info`, etc.
I don't like `annotations`, but I could live with that :)
About `annotate(foo='bar')`: if you intend to use it as a decorator, it would not work in my case because the value depends on the params (see the capnpy example below)